### PR TITLE
chore(github): Adds Contributing guide, issue template, and pull request templates.

### PR DESCRIPTION
**Pull request by markrickert**
I used the `.github` folder on `ignite` as a template for this.

### CODE REVIEW

**Comment by markrickert** [comment]
Blocked by CI until #1442 is merged.

**Review by joshuayoes — APPROVED** [review]
Sick! Nice quality of life 🔥

**Pull request merged**
